static/usage/v7/toast/layout/angular/example_component_ts.md

```diff
@@ -0,0 +1,15 @@
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-example',
+  templateUrl: 'example.component.html',
+})
+export class ExampleComponent {
+  public toastButtons = [
```

Comment:
```suggestion
 toastButtons = [
```

Angular style guides remove the explicit use of `public` on variables/functions.